backend/SwaggerRestApi/SwaggerRestApi/Controllers/CurrentUserController.cs

```diff
@@ -54,5 +54,18 @@ public async Task<ActionResult> UpdateUser([FromBody] UserUpdate user)
             int userId = Convert.ToInt32(userIdString);
             return await _userlogic.EditUser(user, userId);
         }
+
+        [HttpPost("refresh")]
```

Comment:
```suggestion
        [HttpPost("refresh")]
        [Authorize(Roles = "Admin, Operator, User")]
```

Tænker også denne route skal være authorized